### PR TITLE
Document the process of migrating from Replicated to YBA Installer on a different VM

### PR DESCRIPTION
**Pull request by iSignal**

### CODE REVIEW

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/migrate-replicated.md [hunk 1]
```diff
@@ -90,3 +92,17 @@ If Replicated is using HTTP, you need to remove the standbys and delete the HA c
 1. [Configure HA on the updated instances](../../administer-yugabyte-platform/high-availability/#configure-active-and-standby-instances).
 
 [Failovers](../../administer-yugabyte-platform/high-availability/#failover) are possible again after the completion of this step.
+
+### Migrate from Replicated to YBA Installer on a different VM
+
+1. [Install YBA Installer](../install-software/installer/) on a new VM and [configure YBA Installer](../install-software/installer/#configuration-options) appropriately. Note that [Replicated configuration](../install-replicated/#set-up-https-optional) will not be migrated in this method.
+2. Disable any YBA [high availability](../../administer-yugabyte-platform/high-availability/) (HA) configured on the Replicated YBA.
+3. Perform a full backup on the Replicated installation. This backup includes prometheus and all existing releases. If the backup is too large, consider cleaning up unused releases through the YBA UI and/or reducing the prometheus retention interval in Replicated settings.
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/migrate-replicated.md:100]: Are your referring to https://docs.yugabyte.com/stable/yugabyte-platform/administer-yugabyte-platform/back-up-restore-yp/

Comment (ddhodge) [reply docs/content/stable/yugabyte-platform/install-yugabyte-platform/migrate-replicated.md:100]: Probably need links to relevant sections for deleting releases and Prometheus retention.

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/migrate-replicated.md [hunk 2]
```diff
@@ -90,3 +92,17 @@ If Replicated is using HTTP, you need to remove the standbys and delete the HA c
...
+4. Stop YBA on the Replicated VM [through the Replicated UI or command line](../..//administer-yugabyte-platform/shutdown/#replicated). This is a critical step, otherwise you could end up with two YBA instances managing the same set of universes, which can lead to severe consequences.
+5. Transfer the full backup from Step 3 to the YBA Installer VM.
+6. Restore the backup on the YBA Installer VM using the command line `sudo yba-ctl restoreBackup --migration <path to backup.tar.gz>`
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/migrate-replicated.md:103]:
```suggestion
6. Restore the backup on the YBA Installer VM using the command line as follows:

```sh
sudo yba-ctl restoreBackup --migration <path to backup.tar.gz>
```
```

**Review by ddhodge — COMMENTED** [review]
Not sure about the order of putting HA between the 2 methods

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/migrate-replicated.md [hunk 3]
```diff
@@ -30,7 +30,13 @@ If you have high availability configured, you need to migrate your instances in
 
 ## Migrate a Replicated installation
 
-To migrate your installation from Replicated, do the following:
+You can migrate your existing Replicated installation to YBA Installer in place on the same VM (recommended). When migrating in place, both your Replicated configuration and the YBA metadata (universes, providers, and so on) are migrated to the YBA Installer format.
+
+Alternately, you can migrate the [Replicated installation to YBA Installer on a different VM](#migrate-from-replicated-to-yba-installer-on-a-different-vm). Only the YBA metadata (universes, providers, and so on) is migrated to the new VM; the Replication configuration is not.
```
Comment (iSignal) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/migrate-replicated.md:35]:
```suggestion
Alternately, you can migrate the [Replicated installation to YBA Installer on a different VM](#migrate-from-replicated-to-yba-installer-on-a-different-vm). Only the YBA metadata (universes, providers, and so on) is migrated to the new VM; the Replicated configuration is not.
```

**Pull request merged**